src/SablierV2LockupTranched.sol

```diff
@@ -199,7 +199,7 @@ contract SablierV2LockupTranched is
         // at this point.
         uint128 streamedAmount = tranches[0].amount;
         for (uint256 i = 1; i < tranches.length; ++i) {
-            // If a tranche's timestamp is equal to the current time, it is considered vested.
+            // If a tranche's timestamp is less than or equal to the current time, it has vested.
```

Comment:
```suggestion
            // If a tranche's timestamp is less than or equal to the block time, it has vested.
```

We should also rename this in the comment since we have renamed `currentTime` to `blockTimestamp`.

Reply:
Good catch. I will fix now.